recipe/meta.yaml

```diff
@@ -81,12 +80,13 @@ outputs:
         - requests
         - future
         - six
-        - cffi
         - mkl-devel {{ mkl }}   # [x86]
         - libcblas * *_mkl      # [x86]
         - libcblas              # [not x86]
         - liblapack             # [not x86]
         - openblas              # [not x86]
+        - libgomp   # [linux]
```

Comment:
i was only going to ask about this line with respect to the CUDA builds.

Reply:
ok lets skip the builds this time, and maybe upload them over time later

Reply:
ninja was already removed as a dependency so i htink the main issue is resolved via a repo-data patch

Reply:
Congrats on the move!